README.md

```diff
@@ -0,0 +1,171 @@
+<a name="readme-top"></a>
+
+# Hello Rails React
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+ - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📚 Hello Rails React <a name="about-project"></a>
+
+**Hello Rails React** is a sample project that uses REACT for frontend and Rails for backend 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details> -->
+
+<details>
+<summary>Frontend</summary>
+  <ul>
+    <li><a href="https://www.react.dev/">React</a></li>
+  </ul>
+
+<summary>Backend</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.ruby.org/">Ruby</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Connect rails with react** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+[Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+### Setup
+
+Clone this repository to your local machine using the command:
+```sh
+  git clone https://github.com/dazehinn/hello-rails-react.git
+```
+Open your terminal and navigate to the cloned project directory.
+
+### Install
+
+To setup the environment for this project run the two commands below:
+
+
+```sh
+bundle install
+npm install
+```
+
+
+### Usage
+ 
+To run the project, navigate to the project directory and execute the following command:
+
+```sh
+  rails db:create db:migrate 
+  rails db:seed
+  npm run build
+  rails s
+```
+
+### Run tests
+
+```sh
+No tests available for this project
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Nshanji Hilary Ndzi**
+
+- GitHub: [@dazehinn](https://github.com/dazehinn)
+- Twitter: [@HilaryNdzi](https://twitter.com/hilaryndzi)
+- LinkedIn: [@Nshanji Hilary Ndzi](https://www.linkedin.com/in/nshanji-hilary-ndzi/)
```

Comment:
![image](https://github.com/dazehinn/hello-rails-react/assets/93039359/3fab5e2b-dcfb-4864-8e0a-50d384402e46)

- [ ] It would be nice if you fix the highlighted link in the image above 👀 because right now it is broken 😨 Kindly fix it or remove it if you don't have an account for this link to make your readme descriptive and professional 👍